<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сервер печати чеков</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabInfo">
       <attribute name="title">
        <string>Информация</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupDayCheques">
          <property name="title">
           <string>Чеки за день</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableView" name="tableCheques"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupDayTotals">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Итого за день</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QLabel" name="totalsDaySumField">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="totalsNalField">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelChequeQuant">
             <property name="text">
              <string>Чеков пробито:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="totalsCheqQuantField">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelBeznal">
             <property name="text">
              <string>Безналично:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="totalsBeznalField">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelNal">
             <property name="text">
              <string>Наличными:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="totalsCertField">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelDaySum">
             <property name="text">
              <string>Сумма продаж:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelCert">
             <property name="text">
              <string>Сертификатом:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <spacer name="spacerTotals">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QPushButton" name="printAllBarcodesButton">
             <property name="text">
              <string> Напечатать штрихкоды</string>
             </property>
             <property name="icon">
              <iconset resource="../../res.qrc">
               <normaloff>:/res/img/barcode.png</normaloff>:/res/img/barcode.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Настройки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupPrinterSettings">
          <property name="title">
           <string>Параметры устройства печати</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QGridLayout" name="layoutPrinter">
             <item row="0" column="0">
              <widget class="QLabel" name="labelPrinterType">
               <property name="text">
                <string>Тип принтера</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QComboBox" name="printerTypeSelector">
               <item>
                <property name="text">
                 <string>Системный принтер</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM-порт (ZPL)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Godex BZB-2</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="labelPrinterName">
               <property name="text">
                <string>Имя принтера</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="printerNameSelector">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelPathToPrinter">
               <property name="text">
                <string>Путь до принтера</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QLineEdit" name="printerPathEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>/dev/usb/lp0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupComPortSettings">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Параметры COM порта</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QGridLayout" name="layoutComPort">
             <item row="0" column="1">
              <widget class="QComboBox" name="comPortSelector"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelParity">
               <property name="text">
                <string>Четность</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelPortSpeed">
               <property name="text">
                <string>Скорость</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelPortName">
               <property name="text">
                <string>Порт</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelDatabits">
               <property name="text">
                <string>Биты данных</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comDatabitSelector">
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="labelStopbits">
               <property name="text">
                <string>Стоп-бит</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="comParitySelector">
               <item>
                <property name="text">
                 <string>Нет</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Программно (Xon/Xoff)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Аппаратно</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comSpeedSelector">
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>76800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comStopbitSelector">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1,5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="0">
              <spacer name="spacerComPort">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupPaperSettings">
          <property name="title">
           <string>Параметры печати чеков</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="labelPaperSize">
             <property name="text">
              <string>Ширина бумаги</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="paperSizeSelector">
             <item>
              <property name="text">
               <string>26</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="spacerPaperSize">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="testPrinterButton">
             <property name="text">
              <string>Тест печати</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelPrintLogo">
             <property name="text">
              <string>Печать логотипа</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="printLogoCheckbox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupWebserverSettings">
          <property name="title">
           <string>Параметры web-сервера</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QGridLayout" name="layoutWebserver">
             <item row="2" column="1" colspan="2">
              <widget class="QPushButton" name="webserverStopButton">
               <property name="text">
                <string>Остановить</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelServerStatus">
               <property name="text">
                <string>Статус сервера</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="webserverStartButton">
               <property name="text">
                <string>Запустить</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="serverStatusField">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Неизвестно</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QToolButton" name="serverStatusRefreshButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../res.qrc">
                 <normaloff>:/res/img/refresh.png</normaloff>:/res/img/refresh.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelServerPort">
               <property name="text">
                <string>Порт</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QSpinBox" name="webserverPortSelector">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>65535</number>
               </property>
               <property name="value">
                <number>8080</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="saveSettingsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Сохранить все настройки</string>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/res/img/settings-save.png</normaloff>:/res/img/settings-save.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="reloadSettingsButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Отменить изменения</string>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/res/img/settings-cancel.png</normaloff>:/res/img/settings-cancel.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
